src/main/java/ch/jalu/configme/utils/FileUtils.java

```diff
@@ -10,9 +10,9 @@
 /**
  * Utilities class.
```

Comment:
Maybe something like this would be good to keep the documentation in sync with the name: 

```suggestion
 * Class with file utilities.
```